src/pages/Earn/index.tsx

```diff
@@ -91,7 +135,30 @@ export default function Earn() {
       )}
       <TopSection gap="md">
         <AutoColumn>
-          <TokenSelect onTokenSelect={setFilteringToken} token={filteringToken} />
+          <RowStart>
+            <TokenSelect onTokenSelect={setFilteringToken} token={filteringToken} />
```

Comment:
Can we use a consistent height here?

![Screenshot from 2022-05-04 11-05-29](https://user-images.githubusercontent.com/8957844/166777705-033cf9ca-fa3e-48ba-96a0-456cdda55e73.png)